config/initializers/assets.rb

```diff
@@ -1,13 +1,19 @@
 # Be sure to restart your server when you modify this file.
 
-# Version of your assets, change this if you want to expire all your assets.
-Rails.application.config.assets.version = '1.0'
+Rails.application.configure do
 
-# Add additional assets to the asset load path
-Rails.application.config.assets.paths << 'vendor/bower'
+  # Version of your assets, change this if you want to expire all your assets.
+  config.assets.version = '1.0'
 
-# Precompile additional assets.
-# application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-# Rails.application.config.assets.precompile += %w( search.js )
+  # Add additional assets to the asset load path
+  config.assets.paths << 'vendor/bower'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.